apps/test/unit/lib/ui/accounts/ManageLinkedAccountsTest.jsx

```diff
@@ -248,4 +248,83 @@ describe('ManageLinkedAccounts', () => {
     const googleConnectButton = wrapper.find('BootstrapButton').at(0);
     expect(googleConnectButton).to.have.attr('disabled');
   });
+
+  it('in the Maker App, it renders the Google Account as disabled with explanatory tooltip', () => {
+    window.MakerBridge = true;
```

Comment:
this setting will persist on the `window` object after these tests -- we have a couple of `replaceOnWindow` and `restoreOnWindow` helpers that you can use to make sure the state is set back to normal after each test: https://github.com/code-dot-org/code-dot-org/blob/b598fa018617d95ea287a2496b3e07cb84fab6cc/apps/test/unit/code-studio/components/header/ProjectRemixTest.js#L15-L36